### ISSUE TITLE
Consider removing the custom build step

### ISSUE THREAD

**Issue by andiabrudan** [Akebi-Group/Akebi-GC#198]
Hi folks, this issue concerns the custom build step for the `cheat-library` solution in the project.

Personally, I find it more annoying than helpful. I often disable it and build the solution without it.  
It doesn't really do anything extra from what I see. Just launches the executable for the chosen mode and platform.

That's the only thing I have to say about this, really. I don't see how it is of any use. I can run the executable separately.
Maybe more people agree and we get rid of it. Or, if you find it useful, I'd be happy to hear your opinion.

**Comment by Lunairayz** [comment]
afaik devs use it to quick-test something they patched in instead of stealing shit from files, moving somewhere else, bla bla

**Comment by CPP-Callow** [comment]
Yeah, you can run file separately, but I did this for fast debugging. And yeah, it can be kinda annoying, but I didn't find any other ways to do this. I'll think how to solve this problem.
